wikibase.api.js

```diff
@@ -127,12 +127,12 @@ class WikibaseApi {
 		} );
 	}
 
-	getProperty( datatype ) {
+	async getProperty( datatype ) {
 		const envName = `WIKIBASE_PROPERTY_${datatype.toUpperCase()}`;
 		if ( envName in process.env ) {
-			return Promise.resolve( process.env[ envName ] );
+			return await Promise.resolve( process.env[ envName ] );
 		} else {
-			return this.createProperty( datatype ).then( ( propertyId ) => {
+			return await this.createProperty( datatype ).then( ( propertyId ) => {
 				process.env[ envName ] = propertyId;
 				return propertyId;
 			} );
```

Comment:
Now that this is an async function, wouldn't this work just as well and be easier to follow?
```js
if (envName in process.env) {
    return process.env[envName];
} else {
    const propertyId = await this.createProperty(datatype);
    process.env[envName] = propertyId;
    return propertyId;
}
```
But maybe I'm just unfamiliar with all this async/await stuff!

Reply:
^ yes!